libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -19,7 +19,10 @@
  */
 #pragma once
 
-#pragma GCC optimize("O2")
+
+#ifndef HAL_DEBUG_BUILD
```

Comment:
This really ought to be `#if !`, not `#ifndef`

Really annoying when you change a 1 to a 0 and it has no effect.

Reply:
done